nautobot/docs/user-guide/feature-guides/custom-fields.md

```diff
@@ -12,6 +12,11 @@ Navigate to the custom fields page by clicking on **Extensibility -> Custom Fiel
 
 ### Custom Field Attributes
 
++/- 2.0.0
+    Custom field model no longer has a `name` attribute in v2.0. Instead, its pre-existing `label` attribute (unique for all Custom field instances) replaced `name` to be used exclusively for display purposes in the UI.
+    Custom field model's `slug` attribute was renamed to `key` (unique for all Custom Field instances), which contains a GraphQL-safe string used as the key in the underlying custom field data dictionary.
+    These changes do not require users to do any manual work as they are properly handled by data migrations during the upgrade from Nautobot v1.x to v2.0. If you have duplicate Custom field `slug` values in your database pre-migration, some of the Custom field `key` values might be altered to ensure that `key` values for all Custom field instances are unique.
```

Comment:
These still render as a single block in mkdocs - I think maybe what you want is to separate them with blank lines (plus some content edit suggestions):

```suggestion
    CustomField model no longer has a `name` attribute in v2.0. Instead, its pre-existing `label` attribute (unique for all CustomField instances) replaced `name` to be used exclusively for display purposes in the UI.

    CustomField model's `slug` attribute was renamed to `key` (unique for all CustomField instances), which contains a GraphQL-safe string used as the key in the underlying custom field data dictionary.

    These changes do not require users to do any manual work as they are properly handled by data migrations during the upgrade from Nautobot v1.x to v2.0. Note that if you have non-GraphQL-safe `slug` values in your database pre-migration, some of the resulting CustomField `key` values might be altered to ensure that `key` values for all CustomField instances are valid and unique.
```

Reply:
Yeah I tried this and adding blank lines seems to detach the last two paragraphs from the `+/- 2.0.0` block.
<img width="938" alt="Screen Shot 2023-07-20 at 2 10 39 PM" src="https://github.com/nautobot/nautobot/assets/46973263/276e7038-403d-41ee-bd27-5663a5b986c3">

Reply:
Maybe I can break them out into different +/- 2.0.0 blocks.

Reply:
Hmm. It should be possible to include them in a single block. I'll pull down the branch and do some experimenting.

Reply:
Looks fine here:
![image](https://github.com/nautobot/nautobot/assets/5603551/1aeabb63-5a73-4fbd-845a-8022010380e6)
Do you mind if I make the corrections and push to your branch?

Reply:
Ah - you might check whether whatever venv vscode is using has the `mkdocs-version-annotations` python library installed; that preview looks like it didn't understand the annotation.

Reply:
I see I see, will take a look at it. Thanks for the help!